addons/account_edi/models/account_edi_format.py

```diff
@@ -658,6 +658,7 @@ def _retrieve_tax(self, amount, type_tax_use):
             ('amount', '=', float(amount)),
             ('type_tax_use', '=', type_tax_use),
             ('company_id', '=', self.env.company.id),
+            ('price_include', '=', price_include),
```

Comment:
This is changing the behavior quite a lot and could break things

It would be safer to have the default value for `_retrieve_tax` to `None` and only add the domain if `price_include is not None`

Reply:
It seems like you are not using this anymore actually